hedera-node/hedera-token-service/src/main/java/com/hedera/node/app/service/token/AliasUtils.java

```diff
@@ -200,11 +206,21 @@ public static Key asKeyFromAliasPreCheck(@NonNull final Bytes alias) throws PreC
     @Nullable
     public static Key asKeyFromAliasOrElse(@NonNull final Bytes alias, @Nullable final Key def) {
         requireNonNull(alias);
+        // If the alias is an evmAddress we don't need to parse with Key.PROTOBUF.
+        // This will cause BufferUnderflowException
+        if (!isAliasSizeGreaterThanEvmAddress(alias)) {
+            return null;
```

Comment:
Should this be `return def` instead? Since the method name is `...OrElse`?

Reply:
yes agreed Matt. Thanks . Fixed